lib/src/models/google_play_services_availability.dart

```diff
@@ -3,8 +3,11 @@ import 'package:meta/meta.dart';
 /// Indicates possible states of the Google Api Services availability.
 class GooglePlayServicesAvailability {
   const GooglePlayServicesAvailability._(this.value);
+
+  /// Returns the value of the status as a string of the given status integer
   factory GooglePlayServicesAvailability.byValue(int value) => values[value];
 
+  ///Only used for testing purposes
```

Comment:
nit: Same for this comment

```suggestion
  /// Creates an instance of the [GoogleApiAvailability] class. This 
  /// constructor is exposed for testing purposes only and should not be used 
  /// by clients of the plugin as it may break or change at any time. 
```